python/az/aro/azext_aro/tests/latest/unit/test_validators.py

```diff
@@ -3,13 +3,19 @@
 
 from unittest.mock import Mock, patch
 from azext_aro._validators import (
-    validate_cidr, validate_client_id, validate_client_secret, validate_cluster_resource_group, validate_outbound_type,
-    validate_disk_encryption_set, validate_domain, validate_pull_secret, validate_subnet, validate_subnets,
-    validate_visibility, validate_vnet_resource_group_name, validate_worker_count, validate_worker_vm_disk_size_gb, validate_refresh_cluster_credentials,
-    validate_load_balancer_managed_outbound_ip_count
+    validate_cidr, validate_client_id, validate_client_secret,
```

Comment:
nit: I would make them one by one line. This would make diff cleaner.

```suggestion
    validate_cidr,
    validate_client_id,
    validate_client_secret,
```

Reply:
I made this change globally across our imports for consistency's sake.